samples/04-file-transfer/provider/config.properties

```diff
@@ -1 +1,2 @@
-edc.samples.04.asset.path=/path/to/file
\ No newline at end of file
+edc.samples.04.asset.path=/home/ronja/Desktop/test-document.txt
```

Comment:
probably not something we want in github, right?

Reply:
Missed that, removed it now.